inference-engine/tests/functional/inference_engine/pdpd_reader/read_pdpd_model_test.cpp

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <gtest/gtest.h>
+#include <set>
+#include <string>
+#include <fstream>
+
+#include <ie_blob.h>
+#include <ie_core.hpp>
+#include <file_utils.h>
+#include <ngraph/ngraph.hpp>
+#include <ngraph/opsets/opset8.hpp>
+#include <ngraph_functions/utils/ngraph_helpers.hpp>
+
+TEST(PDPD_Reader_Tests, ImportBasicModelToCore) {
+    auto model = std::string(PDPD_TEST_MODELS) + "relu.pdmodel";
```

Comment:
I propose to compare friendly name. Or doesn't it exist in the original model?

Reply:
You mean operation names? Original model doesn't have them, it has only tensor names, we assign first output tensor name as friendly name of operation.

Reply:
I think we need to check tensor and operation names